pages/collection_space_pages.rb

```diff
@@ -216,6 +216,12 @@ def wait_for_title(title_prefix)
     wait_until(Config.medium_wait) { page_title == "#{title_prefix} | CollectionSpace" }
   end
 
+  # @param [String] condition
+  # @return [boolean]
+  def title_bar_field_present?(field)
```

Comment:
@Seanhaoxiang - is this checking whether or not the search parameter is shown above the results?  If so, line 37 does the same thing in https://github.com/cspace-deployment/qa-automation/blob/master/pages/search_results_page.rb